src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -147,7 +155,7 @@ internal static bool VerifyRestoreOutput(RestoreRequest request)
                 
             }
 
-            if (!VerifyPackagesOnDisk(request))
+            if (cacheFile.HasAnyMissingFiles)
```

Comment:
I had originally suggested the change `AnyPackagesMissing` -> `HasAnyMissingPackages` because it's a Boolean property.  I notice that it's now "files" instead of "packages".  I assume that's to better align with the `ExpectedFilePaths` property.

The `ExpectedFilePaths` property is vague.  I'm not sure how intentional that is.  Does the collection contain only package file paths or can the collection contain file paths to other things?

I pose this question, because from just reading this code there's seemingly a potential mismatch in expectations.  If the collection can contain non-package file paths and the only thing that's missing is a non-package file, then the message below would be misleading.

If the collections can only contain package file paths (and not target file paths or whatever) then it might make sense to rename:

* `ExpectedPackageFilePaths` -> `ExpectedPackageFilePaths`
* `HasAnyMissingFiles` -> `HasAnyMissingPackageFiles`

What do you think?

Reply:
Yeah this is a product of an evolving design.  Originally I was going to store every last file that needs to exist in the list.  But then I looked at the code and it made sense to just calculate the path to the assets file, `.g.props`, `g.targets`, and lock file rather than store them in this list.  So the list became just packages.  But I agree, I'll rename it so its name reflects what it stores